types/hapi/tsconfig.json

```diff
@@ -18,80 +18,51 @@
     },
     "files": [
         "index.d.ts",
```

Comment:
why are not the new files listed here?

Reply:
@mhegazy All the new files are in ```index.d.ts```. Example:

```
/** PLUGIN */
export * from './definitions/plugin/plugin';
export * from './definitions/plugin/plugin-registered';

/** REQUEST */
export * from './definitions/request/request';
export * from './definitions/request/request-auth';
...
```

Should I move to ```tsconfig.json```? Thanks for the review.

Reply:
I think having one `index.d.ts` file referencing the other files is cleaner 😄